### PR TITLE
Remove app archive when undeploying

### PR DESCRIPTION
**Pull request by cdonati**

### CODE REVIEW

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/1702/

File: AppController/djinn.rb [hunk 1]
```diff
@@ -1613,6 +1613,8 @@ def stop_app(app_name, secret)
             Djinn.log_warn("(stop_app) got exception talking to " +
               "zookeeper: #{e.message}.")
           end
+          # Prevent future deploys from using the local app archive.
+          FileUtils.rm_rf("#{PERSISTENT_MOUNT_POINT}/apps/#{app_name}.tar.gz")
```
Comment (obino) [inline AppController/djinn.rb]: is there a need to remove also the /var/apps/#{app_name} part? or this is taken care independently?

Comment (cdonati) [inline AppController/djinn.rb]: It's done at the beginning of this function.

**Comment by obino** [comment]
OCD run https://ocd.appscale.com:8080/job/Daily%20Build/1703/

**Pull request merged**